metadata.rb

```diff
@@ -6,5 +6,5 @@
 long_description 'Installs/Configures deploy-user'
 version '1.4.2'
 
-depends 'ssh_known_hosts', '~> 0.2'
+depends 'ssh_known_hosts', '~> 4.0'
```

Comment:
for reference, I had requested it to either be called a different name, or for the work to be attempted to be ported to the official `ssh_known_hosts`. Having it called this name causes confusion and chef incompatibilities (latter if for some reason another cookbook brought in the official one)